### ISSUE TITLE
ODK Briefcase CLI alternative for pulling forms from Ona?

### ISSUE THREAD

**Issue by libcthorne** [onaio/onadata#2234]
https://github.com/getodk/docs/pull/1454 implies that ODK Briefcase is no longer being maintained yet still seems to be recommended by Ona (https://help.ona.io/knowledge-base/how-do-i-use-odk-briefcase/)

Is there an endorsed alternative to ODK Briefcase CLI for pulling and decrypting forms programatically from Ona?

Currently I'm using Briefcase CLI as follows:
```
java -jar briefcase.jar \
  --pull_aggregate \
  --start_from_last \
  --odk_url https://ona.io/example \
  --odk_username example_user \
  --odk_password example_pw \
  --storage_directory example_storage

java -jar briefcase.jar \
  --export \
  --overwrite_csv_export \
  --pem_file example.pem \
  --form_id example_form \
  --storage_directory example_storage \
  --export_directory example_export \
  --export_filename example.csv
```

**Comment by ukanga** [comment]
Yes, ODK Briefaces is no longer maintained but the API it utilizes will still be supported for a long while so we expect the existing versions of ODK Briefcase to work as they have always worked. Your existing CLI workflow should still work.

However, there are other ways to access data currently via the [API](api.ona.io/api). This ranges from using the `/forms`and `/data` API endpoints and the different export formats. It should be possible to put some tools in a CLI environment to use in order to access and download your data. Your current workflow will change.

For now, and especially for those using encrypted forms, we would still recommend ODK Briefcase - the last version released - as we work on other server-side solutions for that specific use case.
